cluster/etcd.go

```diff
@@ -51,7 +51,7 @@ func (c *Cluster) SnapshotEtcd(ctx context.Context, snapshotName string) error {
 		log.Infof(ctx, "[etcd] Finished saving snapshot [%s] on all etcd hosts", snapshotName)
 	}
 
-	if c.Services.Etcd.BackupConfig.S3BackupConfig == nil {
+	if c.Services.Etcd.BackupConfig != nil && c.Services.Etcd.BackupConfig.S3BackupConfig == nil {
```

Comment:
I think `if c.Services.Etcd.BackupConfig == nil || c.Services.Etcd.BackupConfig.S3BackupConfig == nil` is technically the right check here, but I probably should've caught that in the original PR so I'm not going to hold this one up.